tests/components/modbus/test_init.py

```diff
@@ -862,3 +862,20 @@ async def test_integration_reload(
             async_fire_time_changed(hass)
             await hass.async_block_till_done()
     assert "Modbus reloading" in caplog.text
+
+
+@pytest.mark.parametrize("do_config", [{}])
+async def test_integration_reload_failed(hass, caplog, mock_modbus) -> None:
+    """Test setup fails."""
```

Comment:
Minor tweak - looks good otherwise.
```suggestion
    """Run test for integration connect failure on reload."""
```